packages/backend-core/package.json

```diff
@@ -56,7 +57,6 @@
     ]
   },
   "devDependencies": {
-    "@budibase/types": "^1.0.200-alpha.3",
```

Comment:
How come the move from dev to real dependency?

Reply:
Similar for worker and server, where we already have them in dev and now in both?

Reply:
it was failing to run when retrieving dependencies with `NODE_ENV=production yarn` - this appears to be an actual dependency of the running code, I didn't look into it, just moved it as it was required to run.

Reply:
It shouldn't be, it's only ever used as types. Given that it doesn't fail in the develop / release builds, could it be something that needs changed on the single image build rather than in the source?

Reply:
It could have been related to this being based off develop too, maybe an older version that didn't exist or something. Could try again with these moved back into dev after the merge

Reply:
Yeah, I'll give it another go, it was something to do with Analytics, I thought it shouldn't be needed, but wondered if there was some enumeration that was used in release.